Add solutions to lab 4.2 (#211)
acend/openshift-4-ops-training: content/en/docs/04/02_evictions.md

```

> You now know why it was crucial at the beginning of this lab to create the namespace using `oc create namespace`.

Reviewer: ```suggestion
Now you know why it was crucial to create the namespace using `oc create namespace` at the beginning of this lab.
```

> This command does what it says, it simply creates a namespace but does not respect the default project template, so the LimitRange object was never created.

Reviewer: ```suggestion
This command only creates the namespace but does not respect the default project template. That is why the LimitRange object was never created.
```
